scss/components/input-group.scss

```diff
@@ -174,6 +179,9 @@ $block: #{$fd-namespace}-input-group;
         #{$fd-elements-inputs--text} {
             border-top-right-radius: 0;
             border-bottom-right-radius: 0;
+            &:disabled {
+                opacity: 0.4;
+            }
```

Comment:
Both of these changes look like they should be coming from a mixin.  Also seems like this should apply to the `--before` modifier too.

Reply:
> Although these changes are in the `input-group.scss` file, I think this PR should be more focused on input groups as a whole and then time picker will just inherit the fix. I believe there is already a set of changes queued up to address input groups so maybe this PR gets closed in favor of those?

Ok will do.